dm/syncer/dml.go

```diff
@@ -355,6 +355,8 @@ func genSQLMultipleRows(op sqlmodel.DMLType, dmls []*sqlmodel.RowChange) (querie
 	switch op {
 	case sqlmodel.DMLInsert, sqlmodel.DMLReplace, sqlmodel.DMLInsertOnDuplicateUpdate:
 		return sqlmodel.GenInsertSQL(op, dmls...)
+	case sqlmodel.DMLUpdate:
+		return sqlmodel.GenUpdateSQL(dmls...)
```

Comment:
> // Compared to GenInsertSQL with DMLInsertOnDuplicateUpdate, this function is
// slower and more complex, we should only use it when PK/UK is updated.

I noticed this comment in `GenUpdateSQL`, which indicates we should only use it when updating the primary keys, but it seems we didn't consider it in this PR. Would there be any problems?

Reply:
https://github.com/pingcap/tiflow/blob/5140371b5990c91845f79d16f0b2988785083586/dm/syncer/dml.go#L464-L471

this is already implemented